libavfilter/vf_dnn_processing.c

```diff
@@ -62,6 +62,9 @@ AVFILTER_DEFINE_CLASS(dnn_processing);
 static av_cold int init(AVFilterContext *context)
 {
     DnnProcessingContext *ctx = context->priv;
+    DnnContext *dnn_ctx = &ctx->dnnctx;
+    dnn_ctx->model_outputname = ff_dnn_separate_expression(dnn_ctx, dnn_ctx->outputname_expr,
+                                                           "&", &dnn_ctx->nb_output);
```

Comment:
this patch shows that: dnn_processing is broken by your previous patch, it is not allowed. Please think carefully about how to split/merge the patches.